README.md

```diff
@@ -131,17 +131,16 @@ items, as well as the drag items themselves. A drag item can contain
 either text or an image. You can define custom success and error
 feedback for each item. The feedback text is displayed in a popup
 after the learner drops the item on a zone - the success feedback is
-shown if the item is dropped on the correct zone, while the error
+shown if the item is dropped on a correct zone, while the error
 feedback is shown when dropping the item on an incorrect drop zone.
 
-![Zone dropdown](/doc/img/edit-view-zone-dropdown.png)
```

Comment:
@haikuginger This was the only place where `edit-view-zone-dropdown.png` was referenced; you can remove that file from the repo now.